src/style.css

```diff
@@ -0,0 +1,100 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: "Avenir";
+  font-weight: 200px;
+
+}
+
+html {
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
+}
+
+.form-big {
+  position: relative;
+  min-width: 200vh;
+  margin-top: 20px;
+  background-position: center;
+}
+
+.img {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  height: 19px;
+  width: 19px;
+}
+
+.input-big::placeholder {
+  color: #3d4e61;
+  outline: none;
+}
+
+.input-big {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  width: 100%;
+  height: 70px;
+  font-family: inherit;
+  outline: none;
+}
+
+.input-big:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  outline: none;
+}
+
+.input-big:focus {
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  color: #3d4e61;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  outline: none;
+}
+
+.form-small {
+  position: relative;
+  margin-top: 20px;
+  background-position: center;
+}
+
+.img--small {
+  left: 13px;
+  top: 15px;
+  height: 11px;
+  width: 11px;
+}
+
+.input-small {
```

Comment:
you're duplicating a lot of styles from `input-big`

`img` in this comment https://github.com/mate-academy/layout_search-bar-airbnb/pull/2830#discussion_r1096539334 (in the checklist) is only an example, but that is a general rule (should be applied everywhere)